plugins/restful/RestfulDataProviderDbQuery.php

```diff
@@ -349,25 +349,43 @@ public function update($id, $full_replace = FALSE) {
     // Build the update array.
     $request = $this->getRequest();
     static::cleanRequest($request);
+    $save = FALSE;
+    $original_request = $request;
+
     $public_fields = $this->getPublicFields();
     $fields = array();
-    foreach ($public_fields as $public_property => $info) {
+    foreach ($public_fields as $public_field_name => $info) {
+      if (!empty($info['create_or_update_passthrough'])) {
+        // Allow passing the value in the request.
+        if (!empty($info['create_or_update_passthrough_required']) && !isset($request[$public_field_name])) {
```

Comment:
Right now there is no place in RESTful where the user specifies if the data being submitted is valid to be saved or not.

The only way of doing so is with entities via the usage of _Entity Validator_. I like this pattern, and the fact that RESTful is not opinionated about data validation (beyond calling the _Entity Validator_).

Additionally I think that @RoySegall was working on getting generic data validations for _Entity Validator_, so maybe the required passthrough could be deferred to that.